### ISSUE TITLE
isPlaying() function seems slow

### ISSUE THREAD

**Issue by radiohound** [FrankBoesing/Teensy-WavePlayer#10]
This isn't really an issue, just a question. I am using your library to play very short diphone wav sound files (about 70ms in length). These wav files are played in succession to make a word. Originally I had delay statements between the playing each of the wav files (needed to make the speech sound more accurate). I wanted to use a while playWav1.isPlaying() statement to get rid of my delay() statements. But there seemed to be about 200 ms or more of "delay" when I used the isPlaying() statement. I thought that the isPlaying() function would have been a bit faster.  

For most people, this extra delay would not be an issue. I was wondering if you had a better way to quickly find out when a sound file had ended, or if there is a way to make the isPlaying() function faster?

Thank you

**Comment by FrankBoesing** [comment]
The only line in IsPlaying is   return (state == STATE_RUNNING);
There's nothing that can take 200m or more.

**Comment by radiohound** [comment]
Thank you Frank. I will do some more testing to see where my issue might be.

**Comment by radiohound** [comment]
I tested the timing of my code using elapsedMillis. If I am using the isPlaying() feature correctly, the elapsed millis are about 415ms for the S.wav to play and about 585ms for the OH.wav to complete playing. When I tested the codes without the while statements, and only a delay(75), I got 75ms for elapsed millis, which makes sense. The phoneme sounds are very short, in the 20ms range or so. 

The code I was using is here: https://github.com/radiohound/Teensy-WavePlayer/tree/main/examples/Speech-isPlaying

Thank you.
